packages/metamask/src/index.ts

```diff
@@ -57,7 +57,14 @@ function metamask({
         const appLogoUrl = `data:image/svg+xml;base64,${base64}`
         const imports = await importPromise
 
+<<<<<<< HEAD
```

Comment:
more merge conflicts

Reply:
Ahh good catch not sure why that didnt make the initial commit but should be gtg now!